pl_bolts/datamodules/sklearn_datamodule.py

```diff
@@ -6,11 +6,12 @@
 from pytorch_lightning import LightningDataModule
 from torch.utils.data import DataLoader, Dataset
 
+from pl_bolts.utils.warnings import warn_missing_pkg
+
 try:
     from sklearn.utils import shuffle as sk_shuffle
 except ModuleNotFoundError:
```

Comment:
```suggestion
_SKLEARN_AVAILABLE = importlib.util.find_spec("sklearn") is not None
if not _SKLEARN_AVAILABLE:
    from sklearn.utils import shuffle as sk_shuffle
else:
```
lest add this replacement everywhere, in another PR :]

Reply:
Sure, let's do this in the ongoing PR #338.